content/php/concepts/math-functions/terms/min/min.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: 'min()'
+Description: 'Returns the lowest value out of a group or array of values'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+  - 'Web Development'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`min()`** function returns the lowest value out of a group or array of values.
+
+## Syntax
+
+```pseudo
+$lowest = min($value1, $value2, ...$values);
+```
+
+or
+
+```pseudo
+$values = array($n1, $n2, ...n);
+$lowest = min($values);
+```
+
+The `values` can be of any type. The min() function will use the [standard comparison rules](https://www.php.net/manual/en/language.operators.comparison.php), so if a string value is compared to an integer, the string value is considered 0. 
+
+If multiple string values are added, then they will be compared alphanumerically. The returned value will not be converted and if two values are identical, the first in the order will be returned.
+
+If the parameter is an array, the lowest value of the array will be returned.
```

Comment:
Great explanation for the syntax above! It would be helpful to refer to the syntax exactly as is to avoid confusion. For example say `$values` instead of `value`. In addition, place any possible code snippets in backticks (`min()`, `0`)

You can use bullet points to go over each line in the syntax section as well

Reply:
Excellent. Did the changes and will commit them. Thanks! :)